gitlab/datadog_checks/gitlab/data/conf.yaml.example

```diff
@@ -7,89 +9,124 @@ instances:
     #
   - gitlab_url: <GITLAB_URL>
 
+    ## @param api_token - string - optional
+    ## A Gitlab API token for this instance.
+    ## Used to collect the Gitlab version.
+    #
+    # api_token: <API_TOKEN>
+
     ## @param prometheus_url - string - required
-    ## URL of the Prometheus metrics endpoint of gitlab
+    ## The URL where your application metrics are exposed by Prometheus.
     #
     prometheus_url: http://<GITLAB_URL>/-/metrics
 
-    ## @param send_histograms_buckets - boolean - optional - default: true
-    ## The histogram buckets can be noisy and generate a lot of tags.
-    ## send_histograms_buckets controls whether or not you want to pull them.
+    ## @param prometheus_metrics_prefix - string - optional
+    ## <PREFIX> for exposed Prometheus metrics.
     #
-    # send_histograms_buckets: true
+    # prometheus_metrics_prefix: <PREFIX>_
 
-    ## @param send_distribution_counts_as_monotonic - boolean - optional - default: true
-    ## Set send_distribution_counts_as_monotonic to true to send histograms & summary
-    ## counters as monotonic counters (instead of gauges).
+    ## @param health_service_check - boolean - optional - default: true
+    ## Send a service check reporting about the health of the prometheus endpoint.
```

Comment:
Everything not specified in the Gitlab template `gitlab/assets/configuration/spec.yaml ` is inherited from the openmetrics template so that all openmetrics integrations have the same wording. Updating that template directly will change all of the openmetrics integrations.